packages/driver/src/cy/actionability.ts

```diff
@@ -453,6 +455,19 @@ const verify = function (cy, $el, config, options, callbacks: VerifyCallbacks) {
     // the checks and firing the event!
     const retryActionability = () => {
       try {
+        if (callbacks.subjectFn) {
```

Comment:
This is the business end of the PR: `$actionability.verify()` now accepts a subjectFn() callback, to requery the subject between retries. Each command is responsible for passing in a function that resolves the subject; usually a thunk wrapped around `cy.getSubjectFromChain()`, but some commands do additional work on top of that; see each updated command for details.